mscore/editstaff.ui

```diff
@@ -147,7 +147,7 @@
           <item row="1" column="0">
            <widget class="QLabel" name="labelNumOfLines">
             <property name="text">
-             <string>Lines:</string>
+             <string>&amp;Lines:</string>
```

Comment:
Are those mnemonis work here? (Not a rhetorical question ;-), I really don't know)

Reply:
I don't know what they are either. QtDesigner added them so I thought that they must be useful.